src_rebuild/GAME/C/ENVSOUND.C

```diff
@@ -28,6 +28,26 @@ void AddEnvSounds(int level, int time)
 			AddEnvSnd(1, 0x10, 4, 0, -5000, 0x8d9a, -0x2673c, 0x1126a, -0x1efb4);
 			AddEnvSnd(1, 0x10, 4, 0, -5000, 0x8d9a, -0x1368c, 0x1126a, -0x1bb5c);
 			break;
+		case 1:
+			AddEnvSnd(3, 0, 4, 2, 2000, -0x38ba8, -0x41096, -0x38ba8, -0x41096);
+			AddEnvSnd(3, 0, 4, 2, 2000, 0x3d928, -0x1d2cc, 0x3d928, -0x1d2cc);
+		case 5:
+			AddEnvSnd(1, 0, 4, 4, -0xdac, 0x4f236, -0x58dae, 0x52e2c, -0x4ee80);
+			AddEnvSnd(1, 0, 4, 4, -0xdac, 0x52e2c, -0x4fbc8, 0x56734, -0x51338);
+
+			if (level == 5)
+			{
+				AddEnvSnd(1, 0x10, 4, 3, -0x1194, -0x1bcec, -0x2c308, -0x14244, -250000);
```

Comment:
Should convert these to decimal, just hover over the number in Visual Studio.

Reply:
sadly we have only a VS code plugin